tests/source/struct-field-attributes.rs

```diff
@@ -0,0 +1,22 @@
+// #1535
```

Comment:
If the source and target files match exactly (which I think is the case here), you don't need the source file at all (we'll only do an idempotence test).